docs/Guide.md

```diff
@@ -540,6 +541,50 @@ const App = () => {
 
 Note that these cookies will only be sent on the first request unless you use the technique above for [setting custom headers on each page load](#Setting-Custom-Headers).
 
+### Page navigation gesture and button support
+We can provide support for conventional page navigation for both Android and iOS.
```

Comment:
It would be nice to specify here that you're talking about the forward/back swipe gestures on iOS and the hardware back button/gesture on Android.